seqr/views/utils/terra_api_utils.py

```diff
@@ -55,20 +55,10 @@ def anvil_enabled():
 
 
 def is_google_authenticated(user):
-    if not google_auth_enabled() or not hasattr(user, 'social_auth'):
-        return False
-
-    social = _safe_get_social(user)
-    if social and social.extra_data:
-        return social.extra_data.get('access_token', '') != ''
-
-    return False
+    return bool(_safe_get_social(user))
```

Comment:
Does `is_google_authenticated(user)` need to ensure the `user` has logged in? If yes, we may still need to do the token information test.

Reply:
no, it needs to test that they have ever logged in with google, as I stated in the PR description

Reply:
also, again, only with anvil enabled is there ever a token. Currently, logged in users do not have a token. Adding this sort of thing is out of the scope of this PR

Reply:
Oh, yes. No token is needed for local google login.